src/Microsoft.Content.Build.MerfToSDPForJava.SDP.DataContracts/Merf/ArticleItemYaml.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.DataContracts.Merf
```

Comment:
let's use full name "ManagedReference" instead of "Merf"

Reply:
updated